pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -673,17 +744,28 @@ class Build {
         context.node('master') {
             context.stage("installer") {
                 switch (buildConfig.TARGET_OS) {
-                    case "mac": context.sh 'rm -f workspace/target/*.pkg workspace/target/*.pkg.json workspace/target/*.pkg.sha256.txt'; buildMacInstaller(versionData); break
-                    //case "linux": buildLinuxInstaller(versionData); break  // Use the new rpm method
-                    case "windows": buildWindowsInstaller(versionData); break
-                    default: break
+                    case "aix":
+                        buildLinuxInstaller(versionData)
+                        break
+                    case "mac":
+                        context.sh 'rm -f workspace/target/*.pkg workspace/target/*.pkg.json workspace/target/*.pkg.sha256.txt'
+                        buildMacInstaller(versionData)
+                        break
+                    case "linux":
+                        buildLinuxInstaller(versionData)
+                        break
+                    case "windows":
+                        buildWindowsInstaller(versionData)
+                        break
+                    default:
+                        break
                 }
 
                 // Archive the Mac and Windows pkg/msi
                 // (Linux installer job produces no artifacts, it just uploads rpm/deb to the repositories)
                 if (buildConfig.TARGET_OS == "mac" || buildConfig.TARGET_OS == "windows") {
                     try {
-                        context.sh 'cd workspace/target/ && for file in $(ls *.tar.gz *.pkg *.msi); do sha256sum "$file" > $file.sha256.txt ; done'
+                        context.sh 'cd workspace/target/ && for file in $(ls *.tar.gz *.pkg *.msi); do sha256sum "$file" > "$file".sha256.txt ; done'
```

Comment:
Does this fix those files we were seeing named `.sha256.txt`?
Should it be upstreamed?